assets/css/need-a-sweet-heart.css

```diff
@@ -0,0 +1,49 @@
+.need-a-sweet-heart-wrapper{
+    height:80vh;
+    width:100%;
+    display: flex;
+    align-items: center;
+    background-image: url('../images/the-dog-white-and-black.jpg');
+    background-size: cover;
+    background-position-y:50%;
+    justify-content: flex-end;
+}
+.need-a-sweet-heart-box{
+    padding:10%;
```

Comment:
Do wywalenia. W tym projekcie inaczej ustawiamy marginesy. Trzymaj się konwencji

Reply:
od tego masz inner wrapper

W ten sposób robisz w projekcie syf.
Zmienisz inner wrapper w CSSach kiedyś, wszystkie sekcje ładnie się poprawią, a to zostanie babol taki

Reply:
poprawione